templates/assetComponentTemplate.txt

```diff
@@ -0,0 +1,89 @@
+{% macro memberTable(inputMembers, parentName="", linkDescription=true) -%}
```

Comment:
For this and the other templates, Jinja recommends `.html.jinja` as an extension for template files:
https://jinja.palletsprojects.com/en/3.0.x/templates/#template-file-extension

It think it would be more understandable to use their extension instead of `.txt`